packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountsYouManageHomePage.tsx

```diff
@@ -0,0 +1,126 @@
+import { useCallback, useContext, useEffect } from 'react'
+
+import {
+  useApproveManagedAccount,
+  useGetManagedAccounts,
+  useRemoveManager
+} from '@audius/common/api'
+import { Status, UserMetadata } from '@audius/common/models'
+import { accountSelectors } from '@audius/common/store'
+import { Box, Divider, Flex, Text, TextLink } from '@audius/harmony'
+
+import LoadingSpinner from 'components/loading-spinner/LoadingSpinner'
+import { ToastContext } from 'components/toast/ToastContext'
+import { useSelector } from 'utils/reducer'
+
+import { AccountListItem } from './AccountListItem'
+import { sharedMessages } from './sharedMessages'
+import { AccountsYouManagePageProps, AccountsYouManagePages } from './types'
+const { getUserId } = accountSelectors
+
+const messages = {
+  takeControl:
+    'Take control of your managed accounts by making changes to their profiles, preferences, and content.',
+  noAccounts: 'You don’t manage any accounts.',
+  somethingWentWrong: 'Something went wrong. Please try again later.'
+}
+
+export const AccountsYouManageHomePage = ({
+  setPage
+}: AccountsYouManagePageProps) => {
+  const userId = useSelector(getUserId) as number
+  const { data: managedAccounts, status } = useGetManagedAccounts({ userId })
+  const [approveManagedAccount, approveResult] = useApproveManagedAccount()
+  const [rejectManagedAccount, rejectResult] = useRemoveManager()
+  const { toast } = useContext(ToastContext)
+
+  const handleStopManaging = useCallback(
+    ({ userId }: { userId: number; managerUserId: number }) => {
+      setPage(AccountsYouManagePages.STOP_MANAGING, { user_id: userId })
+    },
+    [setPage]
+  )
+
+  const handleApprove = useCallback(
+    ({
+      currentUserId,
+      grantorUser
+    }: {
+      currentUserId: number
+      grantorUser: UserMetadata
+    }) => {
+      approveManagedAccount({ userId: currentUserId, grantorUser })
+    },
+    [approveManagedAccount]
+  )
+
+  const handleReject = useCallback(
+    ({
+      currentUserId,
+      grantorUser
+    }: {
+      currentUserId: number
+      grantorUser: UserMetadata
+    }) => {
+      rejectManagedAccount({
+        userId: grantorUser.user_id,
+        managerUserId: currentUserId
+      })
+    },
+    [rejectManagedAccount]
+  )
+
+  useEffect(() => {
+    if (
+      approveResult.status === Status.ERROR ||
+      rejectResult.status === Status.ERROR
+    ) {
+      toast(messages.somethingWentWrong)
+    }
+  }, [toast, approveResult.status, rejectResult.status])
+
+  return (
+    <Flex direction='column' gap='xl'>
+      <Text variant='body' size='l'>
+        {messages.takeControl}{' '}
+        <TextLink href='#' variant='visible'>
```

Comment:
This going to some external link?

Reply:
yeaah i have a ticket out to figure out what this link is supposed to be lol

Reply:
I think we also have an `ExternalLink` component for things that need to open in a new tab.

Reply:
yes however it's not "harmonized" sadly, but what you can do, is `<TextLink external href='blah' .... />` but we should absolutely update external-link. maybe play around with both and see what works

Reply:
sgtm